lightning-tag/basic-implementation.md

```diff
@@ -0,0 +1,56 @@
+# Basic Lightning Tag Implementation 
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+The base version of Lightning Tag will only:
+
+1. Execute data collection rules that you have configured in cooperation with your Lotame account representative
+1. Deploy subscribed sync pixels
+1. Deploy export beacons 
+
+The base verison of Lightning Tag:
+
+```javascript 
+<head>
+  <link rel="preconnect" href="https://tags.crwdcntrl.net">
+  <link rel="preconnect" href="https://bcp.crwdcntrl.net">
+  <link rel="dns-prefetch" href="https://tags.crwdcntrl.net">            
+  <link rel="dns-prefetch" href="https://bcp.crwdcntrl.net">
+  
+<script>
+  ! function() {
+    var lotameClientId = '<lotameClientId>';
```

Comment:
Possibly add a note to indicate to replace <lotameClientId> with your client id in both places.